gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -655,7 +655,8 @@ message TripDescriptor {
     // (in calendar.txt or calendar_dates.txt) is operating within the next 30 days. The trip to be duplicated is
     // identified via TripUpdate.TripDescriptor.trip_id. This enumeration does not modify the existing trip referenced by
     // TripUpdate.TripDescriptor.trip_id - if a producer wants to cancel the original trip, it must publish a separate
-    // TripUpdate with the value of CANCELED. The VehiclePosition.TripDescriptor.trip_id for the new trip must contain
+    // TripUpdate with the value of CANCELED. Trips defined in GTFS frequencies.txt with exact_times that is empty or
```

Comment:
Why not? A frequency based trip can also have a reinforcement right?

Reply:
@skinkie Yes, a true frequency-based trip (exact_times=0) can have a reinforcement trip, but IMHO the `DUPLICATED` mechanism isn't the right way to express it.

I had to think through this a bit, but here are my thoughts - for RT, using the existing GTFS-RT spec you can just add another TripUpdate with the `trip_id`, `start_date`, and `start_time` and that represents a new frequency-based vehicle. RT frequency-based trip instances are materialized on-the-fly (i.e., there isn't a 1:1 match to static GTFS, unlike normal scheduled trips or exact_times=1 trips), so you could run as many vehicles as you want on the route, including re-enforcement trips  (two buses running bumper-to-bumper), and existing consumers of frequency-based RT data should show them.

If you're trying to express a single new re-enforcement frequency trip without real-time info, then you really don't need to do anything as of today - the UI should still read "arriving every 10 minutes", and that should make sense to the user. The [static spec](https://gtfs.org/reference/static/#frequenciestxt) also says you can't have overlapping headways, so this interpretation seems to align.

If you wanted to change the static headway for the entire service period to say buses were expected to come twice as frequently ("arriving every 5 min" instead of 10 min), then the FrequencyUpdates that we've defined in the larger [GTFS-ServiceChanges v3.1](https://bit.ly/gtfs-service-changes-v3_1) draft would work.